### PR TITLE
S2

### PR DESCRIPTION
**Pull request by SlothSan**

### CODE REVIEW

File: src/App.css.map [hunk 1]
```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":[],"names":[],"mappings":"","file":"App.css"}
```
Comment (nico-dev-it) [inline src/App.css.map]: This file should be in gitignore right?

File: src/App.css [hunk 2]
```diff
@@ -1 +1,4 @@
-
+* {
+    box-sizing: border-box;
+    overflow: hidden;
+}
```
Comment (nico-dev-it) [inline src/App.css:5]: Can you add the windows xp font = Tahoma

File: src/Component/TaskBar/StartButton/StartButton.css.map [hunk 3]
```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["StartButton.scss"],"names":[],"mappings":"AAAA;EACE;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;;AAGF;EACE;EACA;;;AAGF;EACE;EACA;EACA;;;AAGF;EACE;EACA;EACA;EACA","file":"StartButton.css"}
```
Comment (nico-dev-it) [inline src/Component/TaskBar/StartButton/StartButton.css.map:1]: same this file isn't supposed to make it to the repo is it?

**Review by nico-dev-it — COMMENTED** [review]
add windows XP font 
css.map files in the repo

File: src/App.css.map (hunk 1 above)
Comment (SlothSan) [reply src/App.css.map]: Is in the gitignore - will delete any files directly from the branch.

File: src/App.css (hunk 2 above)
Comment (SlothSan) [reply src/App.css:5]: Added

File: src/Component/TaskBar/StartButton/StartButton.css.map (hunk 3 above)
Comment (SlothSan) [reply src/Component/TaskBar/StartButton/StartButton.css.map:1]: Done

**Pull request merged**